bokehjs/src/lib/models/tools/inspectors/hover_tool.ts

```diff
@@ -445,11 +445,38 @@ export class HoverToolView extends InspectToolView {
     const swatch_els = el.querySelectorAll<HTMLElement>("[data-swatch]")
 
     const color_re = /\$color(\[.*\])?:(\w*)/
+    const swatch_re = /\$swatch:(\w*)/
 
     for (const [[, value], j] of enumerate(tooltips)) {
-      const result = value.match(color_re)
-      if (result != null) {
-        const [, opts="", colname] = result
+      let tooltipValue = value
+
+      const colorFieldMatch = value.match(color_re)
+      const swatchFieldMatch = value.match(swatch_re)
```

Comment:
even though it is not as common for JS code, we prioritize matching style with the Python codebase and use snake_case for variables throughout the repo

Reply:
updated